### PR TITLE
added propoverMenuDivider and ability to hide arrow from tooltip

### PR DESCRIPTION
**Pull request by brumeregan**
### What changed
Added component PopoverMenuDivider and ability to hide arrow

### Why it changed
According to this https://app.zeplin.io/project/5864e02695b5754a69f56150/screen/5891b59acb4fd20443b5cd8d PopoverMenu should have divider and props for hide arrow

---

- [x] Change is tested
- [x] Change is documented
- [ ] Build is green
- [x] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

File: src/PopoverMenu/PopoverMenu.e2e.js [hunk 1]
```diff
@@ -31,6 +31,22 @@ describe('PopoverMenu', () => {
...
   }
 
+  async function getPageWithDividerMenu({ rtl } = {}) {
+    const storyUrl = createStoryUrl({
+      kind: '7. Tooltips',
+      story: '7.3 Popover Menu',
+      withExamples: true,
+      rtl,
+    });
+    logs.reset();
```
Comment (argshook) [inline src/PopoverMenu/PopoverMenu.e2e.js]: why do you need these `logs` calls?

File: src/PopoverMenu/PopoverMenu.e2e.js [hunk 2]
```diff
@@ -78,6 +94,19 @@ describe('PopoverMenu', () => {
...
   });
 
+  it('should contains divider', async () => {
+    await getPageWithDividerMenu();
+    await waitForVisibilityOf(driver.element());
+    await driver.click();
+    await waitForVisibilityOf(driver.menu.element());
+
+    await browser.wait(
+      waitForVisibilityOf(driver.menu.element(by.css('[data-hook="story-popover-menu-with-divider-item"]'))),
```
Comment (argshook) [inline src/PopoverMenu/PopoverMenu.e2e.js]: use data hook from story settings

File: src/PopoverMenu/PopoverMenu.js [hunk 3]
```diff
@@ -48,6 +49,7 @@ class PopoverMenu extends WixComponent {
     maxWidth: '378px',
     appendToParent: false,
     zIndex: 0,
+    showArrow: true,
```
Comment (shlomitc) [inline src/PopoverMenu/PopoverMenu.js]: please add also a test for this feature

Comment (brumeregan) [reply src/PopoverMenu/PopoverMenu.js]: added test when showArrow is false, so it shouldn't render

**Pull request merged**